flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -716,6 +716,12 @@ public void acknowledgeCheckpoint(
                 "acknowledgeCheckpoint");
     }
 
+    @Override
+    public boolean notifyTaskFinishing(ExecutionAttemptID executionAttemptID) {
+        // TODO: support adaptive scheduler.
```

Comment:
I am wondering how much extra work is needed to support adaptive scheduler.

Can we support adaptive scheduler by re-using the logic from `SchedulerBase#notifyTaskFinishing`?